websites/A/AnimesWatch/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "972647321970016316",
+	}),
+	strings = presence.getStrings({
+		playing: "general.playing",
+		paused: "general.paused",
+		browsing: "general.browsing",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLVideoElement;
```

Comment:
Why is `video` letted?

Reply:
Because anime can be watched on the site

Reply:
Use a constant?

Reply:
okey

Reply:
but I can't assign a value later?